srcjs/utils.js

```diff
@@ -346,3 +346,9 @@ function updateLabel(labelTxt, labelNode) {
   }
 
 }
+
+// https://stackoverflow.com/a/22551342/1583084
+function isIE() {
```

Comment:
Can you move this into browser.js, replacing `browser.isIE`?

And while you're in there, you might as well update `getIEVersion()` to work with IE11.